### DISCUSSION TITLE
Buildings Heating and Cooling - Assumptions

### DISCUSSION THREAD

**Discussion by juliawawrykowicz** [NREL/ReEDS-2.0#102] [Q&A: Data]
Please advise which file contains assumptions as to the heating / cooling demand by buildings. Preferably, residential and commercial buildings separately. 

Thank you.

**Comment by vcelloho** [comment]
The heating/cooling assumptions aren't modeled separately in ReEDS. The load growth data is based upon EIA's AEO analysis. Their heating and cooling assumptions are documented in the Residential and Commercial Demand modules, with additional data available on request.

For electrification scenarios, we use the Electrification Futures Scenario which includes specific end use load changes as building heating/cooling is electrified. Data and documentation from that work can be found here.
https://www.nrel.gov/analysis/electrification-futures.html

But note that electrification scenario is not used in the model by default.
